### PR TITLE
Add description for Mac platform and delete key world function for build.sh

### PR DESCRIPTION
**Pull request by jonnxu**
Add description for Mac platform and delete key world function for build.sh

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ RocketMQ-Client-CPP is the C/C++ client of Apache RocketMQ which is a distribute
 
 - C and C++ API, include both C style and C++ style apis.
 
-- across platform, all features are supported on both windows and linux system.
+- across platform, all features are supported on Windows Mac and Linux platform.
```
Comment (ShannonDing) [inline README.md]: A comma should be added here after "Windows".

Comment (jonnxu) [reply README.md]: ok

File: README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ doc/rocketmq-cpp_manaual_zh.docx
 
 ## Build and Install ##
 
-### Linux platform ###
+### Linux and Mac platform ###
```
Comment (wlliqipeng) [inline README.md]: It would be better to replace Mac with macOS

Comment (jonnxu) [reply README.md]: change to  Linux and Mac OS,   and delete platform

**Pull request merged**